tests/unit/test_toolchain.cpp

```diff
@@ -92,9 +104,11 @@ void TestToolchain(void)
     test_packed_enum();
 }
 
+#ifndef _WIN32
```

Comment:
The problem is that to get the unit test framework to play nice on Windows and show up pretty [in appveyor](https://ci.appveyor.com/project/jwhui/openthread/build/0.1.28/job/bvdmr6rdngpoefu2/tests), I have to test in a fundamentally different way. I don't produce an application, but a DLL, so there is no `main`.